src/cider/nrepl/middleware/debug.clj

```diff
@@ -221,6 +244,7 @@
     locals: Inspect local variables.
     inject: Evaluate an expression and return it.
     eval: Evaluate an expression, display result, and prompt again.
+    stack: Print the current stack, and prompt again.
```

Comment:
Maybe we should name this `stacktrace` to be crystal clear what exactly the command is for.